.github/workflows/deploy.yml

```diff
@@ -113,6 +118,10 @@ jobs:
         GITHUB_ORG: weaveworks-gitops-test
         GITHUB_TOKEN: "${{ secrets.WEAVE_GITOPS_TEST_WEAVEWORKS_WEAVE_GITOPS_BOT_TOKEN }}"
         GITHUB_KEY: "${{ secrets.WEAVE_GITOPS_TEST_WEAVEWORKS_WEAVE_GITOPS_BOT_SSH_KEY }}"
+        GITLAB_ORG: weave-gitops
+        GITLAB_SUBGROUP: weave-gitops-sub
```

Comment:
I left this in there because I am working on a subgroup test but discovered there needs to be some work done to actually create a repo on a subgroup in the tests. I have written up an issue here https://app.zenhub.com/workspaces/weaveworks-foss-603cb6b1d4dff9000e242dc4/issues/weaveworks/weave-gitops/808